api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -393,6 +393,10 @@
      */
     Attribute<String> CONTAINER_JVM_OPTIONS = new Attribute<>(String2String.getInstance());
     /**
+     * The options of dynamic apex logger appender
+     */
+    Attribute<String> LOGGER_APPENDER = new Attribute<>(String2String.getInstance());
```

Comment:
@PramodSSImmaneni @vrozov Added starting of the programmatic logger appender to Apex client. The appender can be passed to the client via Java command line.